src/Donors/Models/DonorModelQueryBuilder.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace Give\Donors\Models;
+
+use Give\Framework\Database\DB;
+use Give\Framework\Models\ModelQueryBuilder;
+
+class DonorModelQueryBuilder extends ModelQueryBuilder
+{
+
+    /**
+     * Get row
+     *
+     * @unreleased
+     *
+     * @return M|null
+     */
+    public function get($output = OBJECT)
+    {
+        $row = DB::get_row($this->getSQL(), OBJECT);
+
+        if ( ! $row) {
+            return null;
+        }
+
+        $row = $this->attachAdditionalEmails($row);
+
+        return $this->getRowAsModel($row);
+    }
+
+    /**
+     * Get results
+     *
+     * @unreleased
+     *
+     * @return M[]|null
+     */
+    public function getAll($output = OBJECT)
```

Comment:
Can you please add more description about why this has to be overloaded right now and how `attachAdditionalEmails` fit in?  There's kind of a lot going on so our future selves will thank us 😄

Reply:
Resolved: 2e44b47f6020537dbbdf104578350dbb92eb7009